applications/frontend/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "frontend",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "start:dev": "webpack-dev-server",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "devDependencies": {
+    "@material-ui/core": "^4.9.7",
```

Comment:
react, redux, materialui and in general all libraries that are used on runtime should fall into "dependencies" rather than "devDependencies"

Reply:
done